container/templates/init/container.j2.yml

```diff
@@ -39,20 +39,19 @@ settings:
   #  display_name:
 
 services: {}
-  # Add your containers here, specifying the base image you want to build from
-  # For example:
-  #
-  # web:
-  #   from: centos:7
-  #   roles:
-  #     - apache-container
-  #   ports:
-  #     - "80:80"
-  #   command: ['/usr/bin/dumb-init', '/usr/sbin/apache2ctl', '-D', 'FOREGROUND']
-  #   dev_overrides:
-  #     environment:
-  #       - "DEBUG=1"
-  #
+  # Add your containers here, specifying the base image you want to build from.
+  # To use this example, uncomment it and delete the curly braces after services key.
+
+  # version: "2"
+  # services:
+  #   web:
+  #     from: "ubuntu:trusty"
```

Comment:
This is copypasta from the [Getting Started Guide](http://docs.ansible.com/ansible-container/getting_started.html). The idea is that the example service produced by init should be identical to the one in the docs. Whether or not they both should be centos is something I have no particular opinion on, as long as the UX is smooth and intuitive.